config.toml

```diff
@@ -1,17 +1,15 @@
-baseURL = "https://amu-oss.github.io/"
-languageCode = "en-us"
+######################## default configuration ####################
+baseURL = "http://examplesite.org/"
```

Comment:
Update baseURL!
Make it `https://amu-oss.github.io/`

Reply:
Done!!